### ISSUE TITLE
[NPC] Matis the Cruel spams Shoot if you have even the slightest distance from him.

### ISSUE THREAD

**Issue by Darmicus** [WoWManiaUK/Redemption#1251]
The NPC Matis the Cruel who patrols in Bloodmyst is bugged to use his Shoot several times a second, quickly killing anyone. This first happened when I accidentally aggroed him while running past him and I found myself dead. Later, at level 28 I returned to wrap up the quest and still died in the same way, it just took a little longer. 

Both of these times this happened on my Human Paladin. It did NOT happen a couple weeks ago when I did on my Night Elf Druid. I also tested it on my Draenei Shaman and can guarantee it only happens when you have a slight range on him. I attacked at a range with a shock and he started firing Shoot several times a second until I approached him as melee at which point he stopped. The entire time he was walking away from me. Whenever I gave him the slightest distance, the Shoot spam started again. Even after confirmation of having captured him, he kept walking away. If I gave him any distance, he still spammed Shoot while walking, only stopping once his model disappeared due to the quest making him vanish.

I am using your client and latest launcher.

How it should work: Not that this needs to be said, but he shouldn't attack 10+ times a second.  

There was a closed issue on this from over a year ago that mentions the same problem.

**Comment by Freakster1** [comment]
Hello Team,

Matis the cruel has some weird range damage when i blinded the npc it still kept hitting me which is a kind of bug.

Quest is doable but for low levels may hit hard, however when disabled should not do damage to player.

Please find attached snapshot.(Blinded npc still got damage)
![matis the cruel](https://cloud.githubusercontent.com/assets/29118759/26743850/b8105258-47e3-11e7-835a-c133c01a1fc7.jpg)

Its Tweaking :)

**Comment by Darmicus** [comment]
Sorry, took me a little but I made a new character, a Gnome Warlock this time, and decided to test some more stuff. Had to level up to an appropriate level, in this case: 23. I tried as normal and the issue continued as expected. I disabled all addons and the issued continued. I sent in my Voidwalker and found that when someone else holds aggro, I can be at whatever range I want and his attacks won't hit me.
